tsconfig.json

```diff
@@ -2,94 +2,94 @@
   "files": [],
   "references": [
     {
-      "path": "packages/babel"
+      "path": "plugins/babel"
     },
     {
-      "path": "packages/state"
+      "path": "lib/state"
     },
     {
-      "path": "packages/vault"
+      "path": "lib/vault"
     },
     {
-      "path": "packages/wait-for-ok"
+      "path": "lib/wait-for-ok"
     },
     {
-      "path": "packages/webpack"
+      "path": "plugins/webpack"
     },
     {
-      "path": "packages/eslint"
+      "path": "plugins/eslint"
     },
     {
-      "path": "packages/mocha"
+      "path": "plugins/mocha"
     },
     {
-      "path": "packages/n-test"
+      "path": "plugins/n-test"
     },
     {
-      "path": "packages/heroku"
+      "path": "plugins/heroku"
     },
     {
-      "path": "packages/upload-assets-to-s3"
+      "path": "plugins/upload-assets-to-s3"
     },
     {
       "path": "packages/cli"
```

Comment:
**suggestion**: i think `core` might be a clearer name than `packages`?

Reply:
would you say the sandbox is 'core'?

Reply:
`sandbox` is its own category really. maybe it should actually become an integration test fixture that we can also use for manual testing?

Reply:
that sounds reasonable. I'll leave it in `core` for this PR as it feels like that requires a bit more thought if that's okay